### ISSUE TITLE
prove does not include lib

### ISSUE THREAD

**Issue by Tux** [tadzik/panda#148]
panda/Tester.pm calls prove using <code>my $cmd    = "$prove-command -e \"$*EXECUTABLE\" -r t/";</code>

to include lib in the call, prove says (as panda states itself in README.md) to include -l:

<code>prove -e perl6 -lrv t/</code>

but prove ignores -l when -e is used, so the final call will be something like <code>prove -e "perl6" -r t/</code> and -Ilib is not seen or used resulting is massive FAIL (at least for some modules).
I'd like to propose to have panda include -Ilib by default:
<pre>
diff --git a/lib/Panda/Tester.pm b/lib/Panda/Tester.pm
index c86630f..741178c 100644
--- a/lib/Panda/Tester.pm
+++ b/lib/Panda/Tester.pm
@@ -17,7 +17,7 @@ method test($where, :$bone, :$prove-command = 'prove') {
 
         if $run-default && 't'.IO ~~ :d {
             withp6lib {
-                my $cmd    = "$prove-command -e \"$*EXECUTABLE\" -r t/";
+                my $cmd    = "$prove-command -e \"$*EXECUTABLE -Ilib\" -r t/";
                 my $handle = pipe("$cmd 2>&1", :r);
                 my $output = '';
                 for $handle.lines {
</pre>

**Comment by tadzik** [comment]
Is there any module where it actually makes a difference? Note the withp6lib() wrapped around that code, which is doing exactly this -- filling in PERL6LIB with appropriate stuff: https://github.com/tadzik/panda/blob/master/lib/Panda/Common.pm#L14

**Comment by FROGGS** [comment]
It is probably about this: http://irclog.perlgeek.de/perl6/2015-04-12#i_10429150

**Comment by tadzik** [comment]
tl;dr there's a bug in prove *and* a precomp problem which seemed like it's the panda issue. It is not, and it's passed along to appropriate parties, so this one I'm closing.
